### PR TITLE
✨ Add mapping for popular Movies from API

### LINKED ISSUE #62: Homescreen
Add mapping for popular Movies from API. 

image

### PR DESCRIPTION
**Pull request by daebert**
fixes #62

### CODE REVIEW

**Review by JGieseWeb — APPROVED** [review]
Good job 🎯

File: src/app/pages/Homescreen/Homescreen.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import MovieCard from '../../components/MovieCard/MovieCard';
 import styles from './Homescreen.module.css';
 import NavBar from '../../components/NavBar/NavBar';
 import usePopularMovies from '../../hooks/usePopularMovies';
+import { pathToFileURL } from 'url';
```
Comment (lmachens) [inline src/app/pages/Homescreen/Homescreen.tsx]: This seems to be unused

File: src/app/pages/Homescreen/Homescreen.tsx [hunk 2]
```diff
@@ -32,10 +33,12 @@ function Homescreen(): JSX.Element {
...
       <h2 className={styles.watchlistTitle}>Watching</h2>
       <div className={styles.watchlist}>
-        <MovieCard imgSrc="/narcos.png"></MovieCard>
-        <MovieCard imgSrc="/deadpool.png"></MovieCard>
-        <MovieCard imgSrc="/annabelle.png"></MovieCard>
-        <MovieCard imgSrc="/toystory.png"></MovieCard>
+        {popularMovies.map((popularMovie) => (
+          <MovieCard
+            key={popularMovie.id}
+            imgSrc={popularMovie.posterPath || ''}
```
Comment (lmachens) [inline src/app/pages/Homescreen/Homescreen.tsx:38]: Maybe find a good placeholder if `posterPath` is not available.

**Review by lmachens — CHANGES_REQUESTED** [review]
(^///^)

File: src/app/pages/Homescreen/Homescreen.tsx (hunk 1 above)
Comment (daebert) [reply src/app/pages/Homescreen/Homescreen.tsx]: Thank you. Done!

**Review by dhnazarenko — APPROVED** [review]
lgtm 🚀

**Review by Dev-dfm — COMMENTED** [review]
lgtm 🚀

**Review by lajanzen — APPROVED** [review]
Looks great!

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

**Review by l-goede — APPROVED** [review]
very good work!

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

**Review by tmh9x — APPROVED** [review]

**Review by roabram — APPROVED** [review]
lgtm

**Review by infoucarsinan — APPROVED** [review]

**Review by J-Bannach — APPROVED** [review]

**Review by JGieseWeb — APPROVED** [review]
lgtm 👍

**Pull request merged**